src/main/java/jenkins/plugins/mattermost/StandardMattermostService.java

```diff
@@ -29,7 +29,7 @@
 	public StandardMattermostService(String endpoint, String roomId, String icon) {
 		super();
 		this.endpoint = endpoint;
-		this.roomIds = roomId.split("[,; ]+");
+		this.roomIds = roomId.split("[,;]+");
```

Comment:
What about this regular expression: `[,;]+ *` (assuming no name starts with a space?)